GOVERNANCE.md

```diff
@@ -72,8 +72,10 @@ membership beyond these rules.
 
 The CTC may add additional members to the CTC by a standard CTC motion.
 
-A CTC member may be removed from the CTC by voluntary resignation, or by
-a standard CTC motion.
+When a CTC member's participation in CTC matters has become minimal for a
+sustained period of time, the CTC will respectfully ask that member to resign
+from the CTC. In addition to voluntary resignation, a CTC member may be removed
+by a standard CTC motion.
```

Comment:
Thanks, James! I intend to change to say that the letter will request that the member *either* indicate an intention to increase participation *or* convert to observer status. I imagine that would achieve the same thing you're going for. Let me know if not.

I'm not doing a whole lot with this until https://github.com/nodejs/node/pull/7744 either lands or is closed. This one is the touchiest of the three recent (one already landed) proposed governance clarifications, so I'm leaving it for last. So...#7744 next, I guess...